### PR TITLE
Add number of membership associations per 10,000 people

### LINKED ISSUE #221: 23. Add number of membership associations per 10,000 people
Pillar: Opportunity-Rich & Inclusive Neighborhoods
Predictor: Social Capital
Metric: Number of membership associations per 10,000 people

- [x] Rename to `count_membership_associations_per_10k` and `count_membership_associations_per_10k_quality`
- [x] Add 2021 data

### PR DESCRIPTION
**Pull request by malcalakovalski**
Resolves #221

### CODE REVIEW

File: 06_neighborhoods/social-capital/social_associations_county.qmd [hunk 1]
```diff
@@ -0,0 +1,167 @@
+---
+title: "Number of membership associations per 10,000 people — County"
```
Comment (rogin123) [inline 06_neighborhoods/social-capital/social_associations_county.qmd:2]: This file looks good!

**Review by rogin123 — CHANGES_REQUESTED** [review]
I think you should be using a 2020 ZCTA -> 2021 census place crosswalk here instead of 2010. You're downloading 2020 and 2021 data from the census api which should be in 2020 vintage geographies. I think the mismatch is causing 1,786 cbp rows to be dropped when you merge it with the crosswalk

File: 06_neighborhoods/social-capital/social_associations_city.qmd [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+  mutate(naics = as.numeric(NAICS2017)) %>%
+  filter(naics %in% naics_codes_to_keep) %>%
+  # 2,926,340 to 28,599 rows
+  select(zip_code, year, orgs = ESTAB) %>%
+  # remove observations with missing data for our orgs variable
+  drop_na(orgs) %>%
+  # no missings (observations still at 28,599)
+  # aggregate the total # of orgs per ZIP
+  summarize(count_orgs = sum(orgs), .by = c("year", "zip_code")) 
+  # 28,599 observations to 16,644 observations
```
[the remark this answers is not in the document]
Comment (malcalakovalski) [reply 06_neighborhoods/social-capital/social_associations_city.qmd]: Yes, it was an old comment. I just updated it and added `tidylog` to make it easier to track changes in number of observations

File: 06_neighborhoods/social-capital/social_associations_city.qmd [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+merged_sa_zip_city <-
+  merged_sa_zip_city %>%
+  mutate(
+    count_zctas_in_place = n(),
+    .by = c(year, place, place_name)
+  )
+```
+
+Create the merged file where the SA numerator (`count_orgs`) is averaged per Place, weighted by the % area of the ZCTA in that Place (`avg_orgs_per_place_weighted_by_area`) and also include total ZCTAs in Place (`zip_total`) & how many of those partially fall outside the Place (`zips_in`)
+
```
[the remark this answers is not in the document]
Comment (malcalakovalski) [reply 06_neighborhoods/social-capital/social_associations_city.qmd:242]: Ah, great catch, thank you! I updated this comment accordingly.

**Review by rogin123 — APPROVED** [review]
Looks good!

File: 06_neighborhoods/social-capital/social_associations_city.qmd [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+)
+```
+
+We can use this ratio to create a data quality tag for the final file. 
+In particular, these values are:
+
+-    Data Quality 1 = 10% or more of the ZIPs fall mostly in the census place
+
+-    Data Quality 2 = less than 10% of the ZIPs fall mostly in the census place
+
```
[the remark this answers is not in the document]
Comment (tinatinc) [reply 06_neighborhoods/social-capital/social_associations_city.qmd:367]: Hi, yes! I believe this was intentional -- we don't always have 3 buckets for quality if it's not needed/does not make sense for the data source

[the remark this answers is not in the document]
Comment (awunderground) [reply 06_neighborhoods/social-capital/social_associations_city.qmd:367]: The original data quality flag was overly generous. The 10% rule was a very low bar. 

I believe @malcalakovalski adopted a process that uses `afact` and `afact2` that aligns with the ACS microdata metrics.

**Pull request merged**